.travis.yml

```diff
@@ -7,20 +7,20 @@ matrix:
   fast_finish: true
   include:
     - os: linux
-      node_js: "7"
-      env: WEBPACK_VERSION="2.2.0" JOB_PART=lint
+      node_js: "10"
```

Comment:
can we ditch travis and get on circle CI?

Reply:
@shellscape let's do this in other PR

Reply:
OK